server/internal/http_server.go

```diff
@@ -623,6 +670,7 @@ func (s *HttpServer) Start() {
 	r.POST("/start", s.handlerStart)
 	r.POST("/stop", s.handlerStop)
 	r.POST("/ping", s.handlerPing)
+	r.GET("/graphs", s.handleGraphs)
```

Comment:
Can't tman dev-server do this?

Reply:
in production mode, there's no dev-server support, but you still need to access the graph list.